runelite-client/src/main/java/net/runelite/client/plugins/combatlevel/CombatLevelOverlay.java

```diff
@@ -106,27 +106,36 @@ private String getLevelsUntilTooltip()
 
 		// create tooltip string
 		StringBuilder sb = new StringBuilder();
-		sb.append(ColorUtil.wrapWithColorTag("Next combat level:</br>", COMBAT_LEVEL_COLOUR));
+		boolean isMeleeCombatDominating = ((attackLevel + strengthLevel) > (Experience.MAX_REAL_LEVEL / 2 + Experience.MAX_REAL_LEVEL));
 
-		if ((attackLevel + strengthLevel + meleeNeed) <= Experience.MAX_REAL_LEVEL * 2)
+		if (client.getLocalPlayer().getCombatLevel() >= Experience.MAX_COMBAT_LEVEL)
 		{
-			sb.append(meleeNeed).append(" Attack/Strength</br>");
+			sb.append(ColorUtil.wrapWithColorTag("Max combat level achieved", COMBAT_LEVEL_COLOUR));
```

Comment:
This is already validated in the `render` method, which is why on release it does not show any tooltip, so this is never hit. I'd recommend just removing this.